tests/meta/upload/scout/conftest.py

```diff
@@ -87,6 +87,12 @@ def fixture_dna_sample_father_id() -> str:
     return "dna_father"
 
 
+@pytest.fixture(name="extra_tumor_sample_id")
+def fixture_extra_tumor_sample_id() -> str:
```

Comment:
For consistancy
```suggestion
@pytest.fixture(name="extra_tumour_sample_id")
def fixture_extra_tumour_sample_id() -> str:
```

Reply:
Agreed, refactoring in local to also change it in test_meta_upload_scoutapi_rna.py.